docs/src/examples/components/Animation/Types/AnimationExampleDelay.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { Animation, Icon } from '@stardust-ui/react'
+
+const AnimationExampleDelay = () => (
+  <div>
+    {'This animation will start after 5 seconds'}
```

Comment:
```suggestion
    This animation will start after 5 seconds
```

Are brackets necessary?

Reply:
Removed.